packages/flutter/lib/src/cupertino/bottom_tab_bar.dart

```diff
@@ -165,24 +279,30 @@ class CupertinoTabBar extends StatelessWidget implements PreferredSizeWidget {
       );
 
     final Color inactive = CupertinoDynamicColor.resolve(inactiveColor, context);
+
+    // TODO: allow dynamic resizing
```

Comment:
https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-checking-in-comments-that-ask-questions

> TODOs should include the string TODO in all caps, followed by the GitHub username of the person with the best context about the problem referenced by the TODO in parenthesis. A TODO is not a commitment that the person referenced will fix the problem, it is intended to be the person with enough context to explain the problem. Thus, when you create a TODO, it is almost always your username that is given.
>
> Including an issue link in a TODO description is required.